Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN apt-get update \
  && apt-get install -y git zlib1g-dev \
  && docker-php-ext-install zip \
  && a2enmod rewrite \
- && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/apache2.conf \
+ && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/sites-enabled/000-default.conf \
```

Comment:
by default `apache2.conf` includes every `sites-enabled/*.conf` file.
So, you don't require edit your `apache2.conf` file